sixtyfps_runtime/rendering_backends/qt/qt_window.rs

```diff
@@ -889,8 +889,8 @@ impl QtItemRenderer<'_> {
                 rect.is_valid()
                     && (rect.x != 0.
                         || rect.y != 0.
-                        || rect.width != target_width
-                        || rect.height != target_height)
+                        || (rect.width - target_width).abs() > 100. * f64::EPSILON
+                        || (rect.height - target_height).abs() > 100. * f64::EPSILON)
```

Comment:
I wonder if we should start using euclid for these comparisons as we start introducing different factors. Concretely I'm thinking of https://docs.rs/euclid/0.22.6/euclid/approxeq/trait.ApproxEq.html , so like this:

```suggestion
                        || !rect.width.approx_eq(&target_width)
                        || !rect.height.approx_eq(&target_height))
```

(this already compiles since we depend on euclid)

What do you think?

Reply:
Sounds great! I'll update that.

Reply:
Done in the latest push: I went in and replaced all the `EPSILON`s I added before:-)

I did so in one patch on top of what I had before.